numpy/_core/src/multiarray/methods.c

```diff
@@ -1005,7 +1005,7 @@ any_array_ufunc_overrides(PyObject *args, PyObject *kwds)
     if (nin < 0) {
         return -1;
     }
-    fast = PySequence_Fast(args, "Could not convert object to sequence");
+    fast = PySequence_Fast(args, "Could not convert object to sequence"); // noqa: borrowed-ref OK
```

Comment:
I *think* this might be effectively private to NumPy?

Reply:
Ok sounds good. I'll leave this comment as is, then.